MSAL/xcconfig/msal__static__framework__ios.xcconfig

```diff
@@ -0,0 +1,13 @@
+#include "msal__framework__ios.xcconfig"
```

Comment:
Should we call this msal__static__lib or something similar?

Reply:
OK. Renamed to static__lib.